README.md

```diff
@@ -1,2 +1,66 @@
-# test-things
-Collection of Things that can be used for testing different IoT protocols, security mechanisms and interaction styles
+# Device Sandbox
+
+Collection of IoT device simulators that can be used for testing and exploration purposes of different protocols and other Web of Things mechanisms.
+The devices are implemented via various programming languages and frameworks.
+The protocols you can currently test are:
+
+- HTTP
+- CoAP
+- MQTT
+- Modbus
+
+## Dependencies
+
+The project has several dependencies. Currently `JavaScript` and `Python` is used for simulating different devices. Every device has its own dependencies and they should be handled elegantly. For that reason, `Node.js` is used for JS code and `poetry` is used for Python code to run the scripts and handle the dependencies.
```

Comment:
I am not sure what you want to say with "elegantly"

Reply:
Fixed at [156d7be](https://github.com/eclipse-thingweb/test-things/pull/1/commits/156d7be2cc5b36e9d31e612a4f83c4abedef38fc)